### DISCUSSION TITLE
List of tables

### DISCUSSION THREAD

**Discussion by nightz123** [lvjr/tabularray#136] [Q&A]
Hello,
I am using the table in my appendix for some tables. But instead of printing "A.1", "A.2", "B.1" etc. I am only getting "1", "2", "1," etc.
Example of a table:

\begin{longtblr}[
        caption={
                Values%
            },
        label={tab:par},%
    ]{
        colspec={
                l
                l% 
                l% 
            },
        rowhead=2,% 
    }
    \toprule
    Parameter               & Value & Unit \\
    \midrule
    test 1 $\sym{zeta}$ & \num{1}&  \si{-} \\
    test1 $\sym{volumen}$ & \num{0,1}&  \si{\cubic\meter} \\
    test2 $\sym{zeta}$ & \num{1,4}&  \si{-} \\
    \bottomrule
\end{longtblr}
%

**Comment by lvjr** [comment]
See issue #126.
